bats/fb-content-katello.bats

```diff
@@ -110,6 +110,8 @@ gpgcheck=0
 priority=1
 enabled=1
 EOF
+  # newer subscription-manager requires centos 6.8 yum, which is available in CR repo
+  tPackageExists centos-release-cr || tPackageInstall centos-release-cr
```

Comment:
I just asked because I'd expect given how hard its been to keep the pipeline green, that centos-release-cr on EL7 would break something else :grin: